From 0a1b4a5955e7cd5b4b0d488f4fb749d32edea9e5 Mon Sep 17 00:00:00 2001
From: Jiaming Shi <jiaming.shih@gmail.com>
Date: Sun, 17 Aug 2025 13:29:53 +0000
Subject: [PATCH 1/2] Set cc via env in go:generate command

---
 build.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.go b/build.go
index 9e46225..4815eed 100644
--- a/build.go
+++ b/build.go
@@ -2,6 +2,6 @@
 // Copyright (C) 2021 Authors of Cilium */
 
 //go:generate sh -c "echo Generating for $TARGET_GOARCH"
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target $TARGET_GOARCH -cc clang -no-strip KProbePWRU ./bpf/kprobe_pwru.c -- -I./bpf/headers -Wno-address-of-packed-member
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target $TARGET_GOARCH -cc $BPF_CLANG -no-strip KProbePWRU ./bpf/kprobe_pwru.c -- -I./bpf/headers -Wno-address-of-packed-member
 
 package main
-- 
2.39.5

